app/code/Magento/UrlRewrite/Test/Mftf/ActionGroup/AdminSearchByRequestPathAndStoreViewActionGroup.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSearchByRequestPathAndStoreViewActionGroup" extends="AdminSearchByRequestPathActionGroup">
+        <annotations>
+            <description>Goes to the Admin URL Rewrite grid page. Searches the grid based on the provided Redirect Path and StoreView name. Validates that the provided Redirect Path, Type and Target Path are present and correct in the grid.</description>
+        </annotations>
+        <arguments>
+            <argument name="redirectPath" type="string"/>
```

Comment:
No need to repeat arguments declaration from parent AG (this row and following two)

Reply:
fixed